WEB_API/app3.py

```diff
@@ -0,0 +1,97 @@
+import fnmatch
+import os
+import uuid
+import json
+import time
+from datetime import datetime
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from flask import Flask, request, jsonify, send_from_directory, abort, render_template
+from DB.database import User, Upload  # Import the User and Upload classes from your database module
+
+load_dotenv()
+
+UPLOADS_FOLDER = os.getenv('UPLOADS_FOLDER_PATH')
+OUTPUTS_FOLDER = os.getenv('OUTPUTS_FOLDER_PATH')
+
+DB_PATH = os.path.join(os.getenv('DB_FOLDER_PATH'), "gpt_explainer.db")
+engine = create_engine(f"sqlite:///{DB_PATH}", echo=True)
+Session = sessionmaker(bind=engine)
+
+# Set the template folder when creating the Flask app instance
+app = Flask(__name__, template_folder='C:\\targilim\\Excellenteam_python_Project\\final-projects-rurosenb\\TEMPLATES')
+
+
+@app.route('/')
+def index():
+    return render_template('upload_form.html')
+
+@app.route('/upload', methods=['POST'])
+def upload():

```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be